Source/ModCompendiumLibrary/ModSystem/Builders/PersonaPortableModBuilder.cs

```diff
@@ -42,7 +44,7 @@ public VirtualFileSystemEntry Build(VirtualDirectory root, string hostOutputPath
             // Get files from CPK
             VirtualDirectory cpkRootDirectory;
 
-            Log.Builder.Trace($"RootOrCpkPath = {config.CpkRootOrPath}");
+            Log.Builder.Trace($"RootOrCpkPath = {nameof(config.CpkRootOrPath)}");
```

Comment:
this will print "RootOrCpkPath = CpkRootOrPath". Im guessing you mixed up what I said.

Reply:
Yup, my bad. Should be good now 👍